coffee_httpd/src/httpd/server.rs

```diff
@@ -73,6 +75,48 @@ async fn coffee_help(
     Ok(body)
 }
 
+#[api_v2_operation]
+#[post("/install")]
+async fn coffee_install(
+    data: web::Data<AppState>,
+    body: Json<HashMap<String, String>>,
+) -> Result<HttpResponse, Error> {
+    let plugin = body.get("plugin").ok_or_else(|| {
+        actix_web::error::ErrorBadRequest("Missing 'plugin' field in the request body")
+    })?;
+
+    let mut coffee = data.coffee.lock().await;
+    let result = coffee.install(plugin, false, false).await;
+
+    match result {
+        Ok(_) => Ok(HttpResponse::Ok().body(format!("Plugin '{}' installed successfully", plugin))),
+        Err(err) => Err(actix_web::error::ErrorInternalServerError(format!(
+            "Failed to install plugin: {err}"
+        ))),
+    }
+}
+
+#[api_v2_operation]
+#[post("/remove")]
+async fn coffee_remove(
+    data: web::Data<AppState>,
+    body: Json<HashMap<String, String>>,
+) -> Result<HttpResponse, Error> {
+    let plugin = body.get("plugin").ok_or_else(|| {
+        actix_web::error::ErrorBadRequest("Missing 'plugin' field in the request body")
+    })?;
+
+    let mut coffee = data.coffee.lock().await;
+    let result = coffee.remove(plugin).await;
+
+    match result {
+        Ok(_) => Ok(HttpResponse::Ok().body(format!("Plugin '{}' removed successfully", plugin))),
```

Comment:
```suggestion
        Ok(_) => Ok(HttpResponse::Ok().body(format!("Plugin '{}' removed successfully", plugin))),
```

Can we log the value that it is inside the `Ok(_)`?